README.md

```diff
@@ -0,0 +1,29 @@
+#central: a... virtual computer* within java
+
+This is my contribution for Hacktoberfest 2019. My initial plans were for other people to join in and contribute, but... 
+that ain't happening I don't think. Welp, it's open source software, right? And it's for a future project I hope to
+ revive. Actually, it technically is the revival of that project. Just without the Discord.
+
+##what is central?
+central is a way of creating virtual "computers" within java. why do I quote "computers" or asterisk them? because
+ they're not computers in a technical sense. they're just objects. they're more like just extremely complicated state
+  machines.
+  
+  they're not even that complicated, the file system sucked though.
+  
+##what's the uses of central?
+Well, it's mainly just for my usage. However, it can be used for other things of a similar context, I guess. Once 
+it's finished (probably AFTER Hacktoberfest, however I'll continue working on it indefinitely), it'll have some sort
+ of modular command system, maybe a developing language so you can write your own applications... there's a lot on
+  the todo list.
+  
+  Basically, it's just a fancy way of making a state machine.
+  
+###To anyone looking at this repo:
```

Comment:
There should be a space before headings 👀

Reply:
I blame intellij, but will fix